Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -943,23 +943,39 @@ RefPtr<CSSValue> consumeCursor(CSSParserTokenRange& range, const CSSParserContex
     return CSSValueList::createCommaSeparated(WTFMove(list));
 }
 
-RefPtr<CSSValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& context)
+RefPtr<CSSPrimitiveValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& context)
 {
     if (args.peek().type() != IdentToken)
         return nullptr;
 
     CSSParserToken token = args.consumeIncludingWhitespace();
+
     AtomString attrName;
     if (context.isHTMLDocument)
         attrName = token.value().convertToASCIILowercaseAtom();
     else
         attrName = token.value().toAtomString();
 
-    if (!args.atEnd())
-        return nullptr;
+    if (args.size() && args.peek().type() == CommaToken)
+        consumeCommaIncludingWhitespace(args);
+
+    String fallback;
+    bool hasFallback = false;
+
+    if (args.size()) {
```

Comment:
You want to do this based on whether `consumeCommaIncludingWhitespace(args);` was successful or not.